### DISCUSSION TITLE
Default behavior for opening folders in the tree has change, why?

### DISCUSSION THREAD

**Discussion by gleydson404** [nvim-tree/nvim-tree.lua#1884] [Q&A]
My nvim-tree is not working as expected in the first opening. I suppose it should be a config, but I don't know if it exactly.

when I run `lvim .` it opens like this:
![image](https://user-images.githubusercontent.com/9268819/210585635-8e79d73b-a049-4a06-94bb-ce329cfb0e71.png)
but, if i "enter" or "l" on folder1:
![image](https://user-images.githubusercontent.com/9268819/210585770-35c49243-7ca6-400a-947f-8fbc8c6e50c1.png)

Before, its shows the files inside the folder inside the tree, not enter the dir. 

I already try: 
`lvim.setup.open_on_setup` 

but I didn't works as expected:
![image](https://user-images.githubusercontent.com/9268819/210586594-06e122d4-f9ea-4d8c-8747-350cc277ae26.png)

when I switch to left panel, I get the behavior as expected. Or even, if I toggle nvim-tree in a current buffer it also works.   

What can I do, to get the first opening like this?:
![image](https://user-images.githubusercontent.com/9268819/210587482-5f8f5fc5-2c68-4c87-898e-90fd32f3d440.png)

**Comment by alex-courtis** [comment]
I don't have a good answer for you. Startup behaviour is quite fragile, mostly due to timing issues. Attempts to resolve issues around startup have resulted in many breakages.

Please see pinned issue #1669 for a proper resolution that is a work in progress.

**Comment by alex-courtis** [comment]
RE the behaviour of Enter or I on the node it sounds like there might be a misconfiguration, as that is not _directly_ related to startup.

I would be most grateful if you could raise a [bug](https://github.com/nvim-tree/nvim-tree.lua/issues/new/choose) with a minimal configuration that reproduces this issue.

Edit: it would also be useful to see the output `g?` when in the first two image states.
